app/helpers/grants-info.js

```diff
@@ -1,19 +1,15 @@
-const GRANT_PERCENTAGE = 40
-const MIN_GRANT = 35000
-const MAX_GRANT = 1000000
-
-const getGrantValues = (projectCostValue) => {
-  const calculatedGrant = Number(GRANT_PERCENTAGE * projectCostValue / 100).toFixed(2)
+const getGrantValues = (
+  projectCostValue,
+  { minGrant, maxGrant, grantPercentage }
```

Comment:
all of these three variables are inside of the grant info object (which you are passing it from index.js) , instead of duplicate them here and give them as arguments of the getGrantValues , you can just give it like this :
const getGrantValues = (projectCostValue, grantInfo) => 
const calculatedGrant = Number(grantInfo.grantPercentage * projectCostValue / 100).toFixed(2)
  const remainingCost = Number(projectCostValue - calculatedGrant).toFixed(2)
  const isEligible = (
    (grantInfo.minGrant <= calculatedGrant) && (calculatedGrant <= grantInfo.maxGrant)
  )
  return { calculatedGrant, remainingCost, isEligible }
}